README.rst

```diff
@@ -96,10 +96,12 @@ For Developers
     $ make   # show all available makefile targets
     $ make virtualenv # create a virtual environment
     $ source .venv/bin/activate # activate the virtual environment
+    $ You may need to install ``mpich`` Run "sudo apt install mpich" # for example, if you using ubuntu
```

Comment:
This is definitely useful information to have. Since this line is meant as a comment I would add a `#` prefix like: 
`$ # You may need to install ...`